tests/components/lastfm/test_sensor.py

```diff
@@ -0,0 +1,55 @@
+"""Tests for the lastfm sensor."""
+import unittest
+from unittest import mock
+from homeassistant.components.lastfm.sensor import LastfmSensor
+from pylast import LastFMNetwork, Track
+
+
+class MockUser:
+    """Mock user object for pylast."""
+
+    def __init__(self, now_playing_result):
+        """Initialize the mock."""
+        self._now_playing_result = now_playing_result
+
+    def get_playcount(self):
+        """Get mock play count."""
+        return 1
+
+    def get_image(self):
+        """Get mock play count."""
+        pass
+
+    def get_recent_tracks(self, limit):
+        """Get mock recent tracks."""
+        return []
+
+    def get_top_tracks(self, limit):
+        """Get mock top tracks."""
+        return []
+
+    def get_now_playing(self):
+        """Get mock now playing."""
+        return self._now_playing_result
+
+
+class TestLastfmSensor(unittest.TestCase):
```

Comment:
Please use test functions, not classes.

Reply:
I'll change it - but curious to know what's the reasoning?

Reply:
test classes can't leverage fixtures.

Reply:
Interesting - Thanks for explaining - I updated the code and the CI should pass now :) 

Are there alternatives to the lovely helpers you get in the `TestCase` for assertions or is it just a case of writing your own truthy assert statements?